pkg/vsphere/disk/disk.go

```diff
@@ -91,13 +106,22 @@ func (d *VirtualDisk) unlock() {
 
 func (d *VirtualDisk) setAttached(devicePath string) error {
 	if d.Attached() {
-		return fmt.Errorf("%s is already attached (%s)", d.DatastoreURI, devicePath)
```

Comment:
This is what I was originally going for but I couldn't think of a clean way to do it. Thanks!